### ISSUE TITLE
could this be usable as a library?

### ISSUE THREAD

**Issue by walksanatora** [DylanSpeiser/Java-6502-Emulator#14]
I am asking around since this looks pretty cool. would there be any way to implement this as a library in eg: a minecraft mod to bring back the glory of redpower2

**Comment by lythd** [comment]
i think ud have to copy the code cause stuff would have to be modified to make it work for minecraft,
and as long as u give permission i think ur allowed to do that, though that does seem like a daunting task lol but id be happy to see this in minecraft

**Comment by DylanSpeiser** [comment]
Yeah, I could see some of the CPU functions being used for a Minecraft mod. Fair warning, though: I wrote that code a long time ago and it's not very good haha

**Comment by Chase22** [comment]
Currently the UI and emulation code are heavily intertwined so making this a library will need pretty heavy refactoring to make sure your minecraft mod start randomly showing java ui windows 😆 

But once that is done there's not much speaking against using this as a library
